src/components/App.js

```diff
@@ -79,6 +88,15 @@ export class App extends Component {
       case 'producer':
       case 'educator':
       case 'admin':
+        // unless the user is a viewer
+        if (userType === USER_TYPE_VIEWER) {
+          return (
+            <>
+              {headerVisible || standalone ? <Header /> : null}
+              <StudentMode />
+            </>
+          );
+        }
         return (
           <>
             <Header />
```

Comment:
I cannot comment lower, but I think `userType` is missing in `mapStateToProps`.

Reply:
Good catch!